composer.json

```diff
@@ -28,11 +24,16 @@
         "predis/predis": "1.*"
     },
     "require-dev": {
-        "phpunit/phpunit": "4.*"
+        "phpunit/phpunit": "6.*"
     },
     "autoload": {
         "psr-0": {
             "MonologCreator\\": ["src/", "tests/"]
         }
+    },
+    "config": {
+        "platform": {
+            "php":"7.0.0"
```

Comment:
Currently this library has a minimum php version of `>= 5.3.3` (line 22 in this file). This config version should match the minimum PHP version.

Reply:
it is at php 7, so that I can use at least a bit newer phpunit version. I think we did this kind of mix in some of our internal projects as well.

Reply:
Hm I'm not a big fan of it and would actually say we should either not update phpunit or update the entire library to that PHP version. There is normally a strict dependency between these two things.

As long as you feel confident that you kept the PHP code itself compatible to PHP 5.3.3 I can live with this change.
(feel free to resolve then)

Reply:
yeah the library code itself is still backward compatible to PHP 5.3